.github/workflows/repolinter.yml

```diff
@@ -1,31 +0,0 @@
-# NOTE: This file should always be named `repolinter.yml` to allow
-# workflow_dispatch to work properly
-name: Repolinter Action
-
-# NOTE: This workflow will ONLY check the default branch!
-# Currently there is no elegant way to specify the default
-# branch in the event filtering, so branches are instead
-# filtered in the "Test Default Branch" step.
-on: [push, workflow_dispatch]
-
-jobs:
-  repolint:
-    name: Run Repolinter
-    runs-on: ubuntu-latest
-    steps:
```

Comment:
Why this action was removed?

Should it be reintroduced?

Reply:
Seems like it was deleted by mistake at some point, I've brought it back.